controllers/helper.js

```diff
@@ -305,20 +305,44 @@ export async function sendUnifiedEmailForBookedBy(
       isSelfBooking ? bookedBy.cardno : null,
       flattenedMap,
       bookedBy,
-      subject,
-      bookingStatus,
-      welcomeMessage
+      bookingStatus
     );
   }
 }
 
+export function getSubject(bookingStatus){
+  if(bookingStatus == BOOKING_STATUS_PENDING){
+    return 'Booking created';
+  }
+  if(bookingStatus == BOOKING_STATUS_CANCEL){
+    return 'Booking cancelled';
+  }
+  return 'Booking confirmed';
+}
+
+export function getWelcomeMessage(bookingStatus,country){
+
+  if (bookingStatus == BOOKING_STATUS_PENDING)
+  {
+
+    const bookingCreate ="Your Bookings were created.";
+    return (country &&
+    country != 'India' ) ? 
+    bookingCreate+' NRIs can make payments for any bookings in pending status at the Research Center upon arrival.'
+    :bookingCreate+" Payment is due within 24 hours to confirm any bookings in pending status.";
+  }
+
+  if( bookingStatus == BOOKING_STATUS_CANCEL){
+    return "We are sorry to inform you that your bookings have been cancelled.";
+  }
+  return 'We are pleased to inform you that your bookings have been confirmed.';
+}
+
```

Comment:
nitpick: let's use standard formatting using what Vandit shared:

https://vitraagvigyaan.slack.com/archives/C08LC6LK7PW/p1751882693202869?thread_ts=1751865605.367089&cid=C08LC6LK7PW

Reply:
You can also enable Format on Save on VsCode if you want.